src/app/[locale]/(authenticated)/read/[code]/layout.tsx

```diff
@@ -38,21 +38,23 @@ export default async function InterlinearLayout({ children, params }: Props) {
 
 interface Language {
   code: string;
-  name: string;
+  english_name: string;
+  local_name: string;
```

Comment:
We generally convert snake_case to camelCase in the SELECT part of the sql statement to be more idiomatic in javascript. Please look through your changes and make sure you are following this convention